nix/pkgs/xcbeautify/default.nix

```diff
@@ -5,14 +5,15 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "xcbeautify";
-  version = "1.1.1";
+  version = "1.4.0";
   arch = if stdenv.hostPlatform.isAarch64 then "arm64" else "x86_64";
 
+
   src = fetchurl {
-    url = "https://github.com/tuist/xcbeautify/releases/download/${version}/xcbeautify-${version}-${arch}-apple-macosx.zip";
+    url = "https://github.com/cpisciotta/xcbeautify/releases/download/${version}/xcbeautify-${version}-${arch}-apple-macosx.zip";
```

Comment:
Why the owner change?

Reply:
The lib author moved the repo in 1.3.1 release, 
ref -> https://github.com/cpisciotta/xcbeautify/releases/tag/1.3.1

Reply:
Okay.